### PR TITLE
111 review sarcophagus UI

### LINKED ISSUE #111: Review Sarcophagus UI
Update the UI for the final step in the embalmer flow to match designs in figma

Image

### PR DESCRIPTION
**Pull request by knoll3**
# Review create sarco table styling 

Adds styling to the table on the page to review sarco creation. 

### Gotchas
- You may need to disconnect from the bundlr and reconnect for the upload price to populate

image

### Designs
image

### CODE REVIEW

File: src/hooks/viewStateFacet/useGetProtocolFeeAmount.ts [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 
-export function useGetProtocolFeeAmount() {
+export function useGetProtocolFeeAmount(): number {
   const networkConfig = useNetworkConfig();
 
   const { data } = useContractRead({
     addressOrName: networkConfig.diamondDeployAddress,
     contractInterface: ViewStateFacet__factory.abi,
-    functionName: 'getProtocolFeeAmount',
+    functionName: 'getProtocolFeeBasePercentage',
```
Comment (gh-23378) [inline src/hooks/viewStateFacet/useGetProtocolFeeAmount.ts:11]: no wonder that wasn't working before

File: src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts [hunk 2]
```diff
@@ -82,15 +85,23 @@ export function useCreateSarcophagus() {
     archaeologistSignatures,
     archaeologistShards,
     arweaveTxIds,
-    currentStage,
+    currentStage
   });
 
+  // generates a random key with which to encrypt the outer layer of the sarcophagus
+  useCallback(async () => {
```
Comment (gh-23378) [inline src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts]: Calling this out for scrutiny during review, I wasn't sure if this was the way to handle key generation and want to make sure it receives some attention

File: src/features/embalm/stepNavigator/hooks/useUploadPrice.ts [hunk 3]
```diff
@@ -27,7 +27,9 @@ export function useUploadPrice() {
       if (!file || !bundlr) return;
       const price = await bundlr.getPrice(file.size);
       if (!price) return;
-      const convertedPrice = bundlr.utils.unitConverter(price);
+      // pads the price slightly to prevent the user from underfunding
```
Comment (gh-23378) [inline src/features/embalm/stepNavigator/hooks/useUploadPrice.ts:30]: @sethhrbek you'd mentioned wanting to do this earlier but I wanted to confirm

File: src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts (hunk 2 above)
Comment (knoll3) [reply src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts]: The private key that encrypts the outer layer is the recipient's private key and is generated during the "Set Recipient" step. Are we changing this?

Comment (gh-23378) [reply src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts]: inner layer should be the recipient private key, this only impacts the key the embalmer generates which gets distributed to archaeologists in shares

**Pull request merged**